Experiments_toolkit.py

```diff
@@ -166,25 +163,37 @@ def preprocess_with_augmentation(DATA, dictionary, pos_key, augmentation_functio
             window = same_window_size(np_sample[:3], size, 10)
```

Comment:
nit: Above in the comment you write "within the keys are the 3D data samples". While the sensor dimensionality is 3, it's likely interpreted as w x h x d